release-notes-2.0.0.md

```diff
@@ -0,0 +1,15 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
+- **SCMOD-8516**: Extend the security hardening of Java base images by disabling further TLS algorithms apart from [SCMOD-8481]
```

Comment:
Open source repos should not have links to JIRA in release notes. If there is open source documentation in the Java base image for what algorithms were disabled you should be pointing to that, for example: https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/disableWeakTlsAlgorithms.patch

There should be one for Java 11 too so make sure you point to the correct one depending on which base image these projects are using.